frontend/model/state.js

```diff
@@ -758,7 +760,7 @@ const actions = {
 
       if (attemptToReprocessMessageID === message.hash()) {
         console.warn(`Successfully re-processed ${attemptToReprocessMessageID}!`)
-        attemptToReprocessMessageID = null
+        attemptToReprocessMessageID = ''
```

Comment:
This would mean that `undefined` is also a valid value, since so-called [Maybe types](https://flow.org/en/docs/types/maybe) in Flow include both `null` and `undefined`.

Reply:
OK, this is fine